README.md

```diff
@@ -13,6 +13,14 @@ It is a "flat" PHP project; individual php files within the `public` folder hand
 
 **[XAMPP](https://www.apachefriends.org/download.html)** provides an easy way to run an Apache web server, MySQL/MariaDB, and PHP on your system.
 
+- Note that XAMPP comes packaged with PHP 7.2. You will need to update to PHP 7.4 or errors will occur when you open the application. To update PHP, do the following:
```

Comment:
I suggest moving these instructions to the wiki and here on the README just put a note saying something like "use PHP 7.4+ (check the wiki if you're having problems)"

Reply:
That makes sense. I will make that change. Thanks!